### PR TITLE
fix pip install

### LINKED ISSUE #1211: Can't install TARDIS with `pip`
## Description

```
pip install git+https://github.com/tardis-sn/tardis
```
... does not work anymore!.

```
ModuleNotFoundError: No module named 'pyne'
```

Please, try to reproduce this bug in a fresh environment with latest TARDIS release and fix.

### PR DESCRIPTION
**Pull request by epassaro**
**Description**

Fixes the `pip install ...` command. Some examples:

- `pip install git+https://github.com/tardis-sn/tardis.git`
- `pip install .` (equivalent to `python setup.py install` )
- `pip install -e .` (equivalent to `python setup.py develop`)

To achieve this I needed to undo some changes made in #1674 related to the `tardis` script. Now that script is considered a pure Python script instead of an entry point.

**Motivation and context**

We had two `scripts` folder before APE 17 migration: 

- the`scripts/` (containing just the `tardis` UNIX script) 
- and `tardis/scripts` (Python entry points)

In #1674 made some changes to the `tardis` script to use it as an entry point. But seems that worked for `python setup.py develop`, but didn't worked for `pip install .` (required by the `conda-forge` packaging system, [see here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=347242&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=691)).

I reverted those changes plus some minor changes to make the script tidier.

You can try the result of `pip install .` on this branch and compare it with `master`.

Fixes #1211

**How has this been tested?**
- [ ] Testing pipeline.
- [x] Other. 

Locally.

**Examples**

**Type of change**

- [x] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

**Review by yuyizheng1112 — APPROVED** [review]

**Comment by yuyizheng1112** [comment]
There is no `.py` extension after `tardis/scripts/tardis`. Is it on purpose?

**Comment by epassaro** [comment]
> There is no `.py` extension after `tardis/scripts/tardis`. Is it on purpose?

Yes. At the header of that file, you can see `#!/usr/bin/env python` which tells the script which interpreter should use. This script is intended to be used by typing `tardis` on the command line (without the extension).

File: tardis/scripts/tardis [hunk 1]
```diff
@@ -1,13 +1,25 @@
 #!/usr/bin/env python
```
Comment (jaladh-singhal) [inline tardis/scripts/tardis:1]: I don't get it how it is related to pip install? 

It seems like it makes tardis runnable via CLI - is that so?

Comment (epassaro) [reply tardis/scripts/tardis:1]: **Some context:**

We had two `scripts` folder before APE 17 migration: 

- the`scripts/` (containing just the `tardis` UNIX script) 
- and `tardis/scripts` (Python entry points)

In #1674 made some changes to the `tardis` script to use it as an entry point. But seems that worked for `python setup.py develop`, but didn't worked for `pip install .` (required by the `conda-forge` packaging system, [see here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=347242&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=691)).

I reverted those changes plus some minor changes to make the script tidier.

You can try the result of `pip install .` on this branch and compare it with `master`.

**Pull request merged**